src/containers/PartyView.js

```diff
@@ -66,28 +66,32 @@ function PartyView({ location }) {
       setPartyName(selected);
       setExistingParty(true);
     }
-    const socket = DungeonService.watchCharacters(ids);
-    socket.onmessage = (event) => {
-      const updatedCharacters = JSON.parse(event.data).sort((a, b) => {
-        if (!a.initiative) {
-          if (!b.initiative) return 0;
-          return 1;
-        }
-        if (!b.initiative) return -1;
-        return b.initiative - a.initiative;
-      });
-      setCharacters(updatedCharacters);
-    };
+    async function fetchCharactersByOwner() {
+      const owned = await DungeonService.getCharactersByOwner();
+      setCharacters(owned);
+    }
+    fetchCharactersByOwner();
+    // const socket = DungeonService.watchCharacters(ids);
+    // socket.onmessage = (event) => {
+    //   const updatedCharacters = JSON.parse(event.data).sort((a, b) => {
+    //     if (!a.initiative) {
+    //       if (!b.initiative) return 0;
+    //       return 1;
+    //     }
+    //     if (!b.initiative) return -1;
+    //     return b.initiative - a.initiative;
+    //   });
+    //   setCharacters(updatedCharacters);
+    // };
```

Comment:
just delete for now, this will be a ghost for too long and we may not implement the same way